Core/source/platform/util.ts

```diff
@@ -55,31 +55,34 @@ export const str_to_hex = (str: string): string => {
     r.push("" + h);
   }
   return r.join('');
-}
+};
 
+/* tslint:disable:no-null-keyword */
 const maxDate = (dates: (Date | null)[]): Date | null => {
-  let res: Date | null = null;
+  let res: Date | undefined;
   for (const date of dates) {
-    if (res == null || (date != null && date > res)) {
+    if (date != null && (res === undefined || date > res)) {
```

Comment:
`!=` and `==` should be disabled with tslint or eslint. (use `!==` and `===` instead). I'm surprised this passes. Can you please enable the rule for it?

Also, this method was refactored in a way that it's not obvious (from code) if it still does the same thing.

Reply:
It should be already enabled, because I've seen it has worked in other places. Not sure why it didn't work here.
I will better restore previous code, since check for null keyword is already disabled.